gui/slick/js/ajax-episode-subtitles.js

```diff
@@ -1,59 +1,189 @@
+        }));
+    }
+
+    $('#manualSubtitleSearchModal').on('hidden.bs.modal', function () {
+        // If user close manual subtitle search modal, enable again all searches
+        enableAllSearches();
+    });
+
     $.ajaxEpSubtitlesSearch = function() {
-        $('.epSubtitlesSearch').on('click', function() {
-            var subtitlesTd = $(this).parent().siblings('.col-subtitles');
-            var subtitlesSearchLink = $(this);
-            // fill with the ajax loading gif
-            subtitlesSearchLink.empty();
-            subtitlesSearchLink.append($('<img/>').prop({
-                src: 'images/loading16.gif',
-                alt: '',
-                title: 'loading'
-            }));
-            $.getJSON($(this).attr('href'), function(data) {
+        $('.epSubtitlesSearch').on('click', function(e) {
+            // This is for the page 'displayShow.mako'
+            e.preventDefault();
+            selectedEpisode = $(this);
+            subtitlesTd = selectedEpisode.parent().siblings('.col-subtitles');
+            // Ask user if he want to manual search subs or automatic search
+            $('#askmanualSubtitleSearchModal').modal('show');
+        });
+
+        $('.epSubtitlesSearchPP').on('click', function(e) {
+            // This is for the page 'manage_subtitleMissedPP.mako'
+            e.preventDefault();
+            selectedEpisode = $(this);
+            subtitlesTd = selectedEpisode.parent().siblings('.col-search');
+            searchSubtitles();
+        });
+
+        $(document).on('click', '#pickSub', function(event){
+            subtitlePicked = $(this);
+            changeImage(subtitlePicked, 'images/loading16.gif', 'loading', 'loading', 16, true);
+            var subtitleID = subtitlePicked.attr('subtitleID');
+            // Remove 'subtitleid-' so we know the actual ID
+            subtitleID = subtitleID.replace('subtitleid-', '');
+            url = selectedEpisode.prop('href');
+            url = url.replace('searchEpisodeSubtitles', 'manual_search_subtitles');
+            // Append the ID param that 'manual_search_subtitles' expect when picking subtitles
+            url += '&picked_id=' + subtitleID;
+            $.getJSON(url, function(data) {
+                // If user click to close the window before subtitle download finishes, show again the modal
+                if (($('#manualSubtitleSearchModal').is(':visible')) === false) {
+                    $('#manualSubtitleSearchModal').modal('show');
+                }
+                if (data.result == 'success') {
+                    changeImage(subtitlePicked, 'images/save.png', 'subtitle saved', 'subtitle saved', 16, true);
+                    // Need to add 1 because of the row header
+                    var removeRow = parseInt(data.release_id) + 1
+                    if (document.getElementById('releasesPP') !== null) {
+                        document.getElementById('releasesPP').deleteRow(removeRow);
```

Comment:
Try not to mix and match vanilla js and jQuery, so maybe use $('#releasesPP') here instead of `document.getElementById`.

I'm not 100% sure this can be done though since I think deleteRow is only applicable for vanilla js selectors. Just check that.